common/src/main/java/com/wynntils/core/webapi/profiles/item/IdentificationProfile.java

```diff
@@ -32,14 +32,14 @@ public IdentificationProfile(IdentificationModifier type, int baseValue, boolean
     }
 
     public void calculateMinMax(String shortId) {
+        isInverted = IdentificationOrderer.INSTANCE.isInverted(shortId);
```

Comment:
Why did you move this?

Reply:
the value was used earlier than calculated

Reply:
Is it, really? Am I blind? It has moved from line 41 to line 35. I see no mention of `isInverted` between those lines. It is refered to in line 43, but the old position was before that as well. There are no method calls between line 35 and 41. I can't see how this change affects anything.

Reply:
Ah, now I get it. I was just really thick. Thanks!